src/premiscale/config/parse.py

```diff
@@ -54,7 +54,11 @@ def configParse(configPath: str) -> Config:
                 sys.exit(1)
 
             # Parse the config into a Config object, now.
-            _config = build_config_from_version(_loaded_config['version']).from_dict(_loaded_config)
+            _config = build_config_from_version(
+                _loaded_config['version']
+            ).from_dict(
+                _loaded_config
+            )
```

Comment:
The longer I stare at this, the more confusing  this call seems.

Reply:
Like, why are we referecning a field contained within the config. Maybe I should split these two methods entirely.

Reply:
Oh actually this makes total sense, it returns a `Type`; this is related to AGENT-38.